tests/decorators/test_short_circuit.py

```diff
@@ -30,49 +31,84 @@
 DEFAULT_DATE = datetime(2022, 8, 17)
 
 
-def test_short_circuit_decorator(dag_maker):
-    with dag_maker(serialized=True):
-
-        @task
-        def empty(): ...
+@pytest.mark.xfail(reason="This test might fail due to serialization issues")
```

Comment:
? what was the failure

Reply:
Hmmm..I removed the `xfail` and now it seems to work (at least running breeze locally).
Let's see if it reproduces in the CI.